script.js

```diff
@@ -1,78 +1,91 @@
-const books = [];
-const titlePlace = document.getElementById('title');
-const authorPlace = document.getElementById('author');
-const form = document.getElementById('form');
+/* eslint-disable */const form = document.getElementById('form');
```

Comment:
- Kindly enable your linters.
**Check the way on how to use ESLint comments** [here](https://eslint.org/docs/2.13.1/user-guide/configuring).

Reply:
Ok, thank you, now the linters are enabled and without errors 😄